app/views/bookings/index.html.erb

```diff
@@ -1,19 +1,51 @@
 <div class="container">
-  <h1 class="m-2">Your bookings.</h1>
+
+  <h1 class="m-2">Your bookings</h1>
   <div class="row justify-content-center">
     <% @bookings.each do |booking| %>
+      <% if booking.user == current_user %>
+       <%# this if checks that the user is the person booking the dress %>
       <% booked_dress = Dress.find(booking.dress_id) %>
         <div class="dress-card text-center mb-3">
           <%= link_to booking_path(booking) do %>
             <%# Image tag to be updated with cloudinary %>
             <%= booked_dress.photos.attached? ? cl_image_tag(booked_dress.photos.first.key) :
             image_tag('https://images.unsplash.com/photo-1557363763-8381968f8353?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80') %>
           <% end %>
+          <p>Booking-ID:<%= booking.id %></p>
+          <% booker = User.find(booking.user_id) %>
+          <p>Booker:<%= booker.last_name%></p>
           <p><%= booked_dress.description %></p>
           <p>$<%= booked_dress.price_per_day %> per day</p>
           <p><%= booking.starting_date %></p>
           <p><%= booking.ending_date %></p>
         </div>
+      <% end %>
     <% end %>
   </div>
+
```

Comment:
I actually did something similar on the profile page earlier today. The users can see their bookings there. Maybe we don't need both? For now perhaps we can merge it and then see where we would rather have this information.

Reply:
Sorry, I should've said something on the slack about making the index and show logic on the html. My bad  :(

Reply:
Your profile looks great. Please use your page. Could we merge it and then you can take any of the logic you think works and then we could even take away the booking_index. That would make things a little easier for routing and testing.
I also, think the logic on the index page would work as one "if else" statement than two "if" statements if you want to use it ^_^